docs/development/maintainers.md

```diff
@@ -103,21 +109,19 @@ This can be done with either,
   ```
   and indicate which commits to take from `main` in the UI.
 
-Then follow steps 1, 2, 5 and 6 from {ref}`making-major-release`.
+Then follow the relevant steps from {ref}`release-instructions`.
 
 ### Making an alpha release
 
-Follow steps 1, 5, and 6 from {ref}`making-major-release`. Name the first
-alpha release `x.x.xa0` and in subsequent alphas increment the final number. For
-the npm package the alpha should have version in the format `x.x.x-alpha.0`. For
-the node package make sure to use `npm publish --tag next` to avoid setting the
-alpha version as the stable release.
-
-If you accidentally publish the alpha release over the stable `latest` tag, you
-can fix it with: `npm dist-tag add pyodide@a.b.c latest` where `a.b.c` should be
-the latest stable version. Then use
-`npm dist-tag add pyodide@a.b.c-alpha.d next` to set the `next` tag to point to the
-just-published alpha release.
+Name the first alpha release `x.x.xa0` and in subsequent alphas increment the
```

Comment:
It seems like we were using `a1` for the first alpha release, which, I guess, is not very important.

Reply:
```suggestion
Name the first alpha release `x.x.xa1` and in subsequent alphas increment the
```